packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -0,0 +1,164 @@
+| `urn:okta:loa:1fa:pwd`   | Password only. Allows one factor authentication that requires the user’s password. | Query | String |
+| `urn:okta:loa:1fa:any`   | Any one factor. Allows one factor authentication with no requirements on which factor. | Query | String |
+| `urn:okta:loa:2fa:any`   | Any two factors. Allows two factor authentication with no requirements on which factors. | Query | String |
+| `phr` <ApiLifecycle access="ie" /> | Phishing-Resistant. Requires users to provide possession factors that cryptographically verify the sign-in server (the origin). Currently, only FIDO2/WebAuthn satisfies this requirement. Because phishing resistance implies [device binding](https://help.okta.com/okta_help.htm?type=oie&id=ext-configure-authenticators), that constraint is selected automatically when `phr` is specified. | Query | String |
+| `phrh` <ApiLifecycle access="ie" /> | Phishing-Resistant Hardware-Protected. Requires that you store keys being used to authenticate in secure hardware (TPM, Secure Enclave) on the device. Currently, only Okta Verify meets this constraint. Because hardware protection implies [device binding](https://help.okta.com/okta_help.htm?type=oie&id=ext-configure-authenticators), that constraint is selected automatically when `phrh` is specified. | Query | String |
+
+## Authentication flow using ACR values
+
+<div class="three-quarter">
+
+![Flow diagram that displays the back and forth between the client, resource server, and the authorization server](/img/auth/step-up-authentication-using-acr-values-flow.png)
+
+</div>
+
+At a high-level, this flow has the following steps:
+
+1. Per your use case, include the `acr_values` predefined parameter value in the authentication request.
+
+2. The authentication scenarios required by the [grant type](/docs/guides/implement-grant-type/authcode/main/) authenticate the user in accordance with the predefined `acr_values` parameter value used in the authorization request.
+
+3. When the authentication flow completes, the authorization server returns an access token and/or an ID token to the client that contains the `acr` claim.
+
+4. The client requests access to the protected resource and presents the new access token.
+
+5. The resource server evaluates the assurance level of the access token against its requirements and then returns the protected resource.
+
+## Add the parameter value to the auth request
+
+The following is an example authorization request using the `urn:okta:loa:1fa:any` predefined `acr_values` parameter value. In this example, the Authorization Code [grant type](/docs/guides/implement-grant-type/authcode/main/) is used.
+
+**Request**
+
+```bash
+https://${yourOktadomain}/oauth2/default/v1/authorize?client_id={clientId}
+&response_type=code
+&scope=openid
+&redirect_uri=http://${yourOktadomain}/authorization-code/callback
+&state=296bc9a0-a2a2-4a57-be1a-d0e2fd9bb601
+&urn:okta:loa:1fa:any
```

Comment:
request missing `acr_value=`

Reply:
got it. will update when I update the other examples next week.